entrypoint.sh

```diff
@@ -30,19 +40,32 @@ if ! APTIBLE_OUTPUT_FORMAT=json aptible apps | jq -e ".[] | select(.handle == \"
   exit 1
 fi
 
+BRANCH="$GITHUB_HEAD_REF"
+if [ -z "$BRANCH" ]; then
+  BRANCH="$GITHUB_REF_NAME"
+fi
+if [ -z "$BRANCH" ]; then
+  echo "Aborting: branch is not set; this shouldn't happen, please contact support: https://www.aptible.com/docs/support"
+  exit 1
+fi
+
+REPO_URL="https://github.com/$GITHUB_REPOSITORY"
+COMMIT_SHA="$GITHUB_SHA"
+COMMIT_URL="$REPO_URL/commit/$COMMIT_SHA"
+COMMIT_DATE=$(git log -1 --date=iso-strict --format="%ad")
+GIT_CONFIG_VARIABLES="APTIBLE_GIT_COMMIT_SHA=$COMMIT_SHA APTIBLE_GIT_REPO_URL=$REPO_URL APTIBLE_GIT_REF=$BRANCH APTIBLE_GIT_COMMIT_URL=$COMMIT_URL APTIBLE_GIT_COMMIT_DATE=$COMMIT_DATE"
+COMMIT_MESSAGE=$(git log -n 1 --pretty=format:'%s%n')
```

Comment:
why is this not included in GIT_CONFIG_VARIABLES?

Reply:
We ran into issues with the commit message contents getting parsed incorrectly when we embedded it inside the `GIT_CONFIG_VARIABLES` so this is what ended up working for us.